core/components/atoms/card/__stories__/style.css

```diff
@@ -0,0 +1,7 @@
+.Cards {
```

Comment:
Please use better specific class names, like `Diagnosis-list`.

Reply:
`Diagnosis-list ` will replace `Cards-list` do I need to change `Cards` as well?